launcher/ui/MainWindow.cpp

```diff
@@ -1826,31 +1827,52 @@ void MainWindow::droppedURLs(QList<QUrl> urls)
             break;
         }
 
-        auto localFileName = url.toLocalFile();
+        auto localFileName = QDir::toNativeSeparators(url.toLocalFile()) ;
         QFileInfo localFileInfo(localFileName);
 
-        bool isResourcePack = ResourcePackUtils::validate(localFileInfo);
-        bool isTexturePack = TexturePackUtils::validate(localFileInfo);
+        auto type = ResourceUtils::identify(localFileInfo);
+
+        // bool is_resource = type;
 
-        if (!isResourcePack && !isTexturePack) {  // probably instance/modpack
+        if (!(ResourceUtils::ValidResourceTypes.count(type) > 0)) {  // probably instance/modpack
```

Comment:
```suggestion
        if (!ResourceUtils::ValidResourceTypes.contains(type)) {  // probably instance/modpack
```

Reply:
`std::set.contains` is only in C++20 aren't we limiting ourselves to c++17?

Reply:
oh
i didn't notice that :clown_face: 

_pretend i never said anything_